e2e/testcafe-devextreme/tests/accessibility/menu.ts

```diff
@@ -40,10 +40,7 @@ const items: Item[] = [
 
 const options: Options<Properties> = {
   items: [items],
-  disabled: [true, false],
```

Comment:
This parameter may affect the availability of attributes and should not be removed.